order-service/src/main/java/com/hoangthh/api/mapper/ProductMapper.java

```diff
@@ -0,0 +1,20 @@
+package com.hoangthh.api.mapper;
+
+import com.hoangthh.api.dto.request.ProductBodyRequest;
+import com.hoangthh.core.domain.ProductBody;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ProductMapper {
+
+    public ProductBody toDomain(ProductBodyRequest productBodyRequest) {
+        ProductBody productBody = new ProductBody();
+        productBody.setProductId(productBodyRequest.getProductId());
+        productBody.setProductName(productBodyRequest.getProductName());
+        productBody.setPrice(productBodyRequest.getPrice());
+        productBody.setQuantity(productBodyRequest.getQuantity());
+
```

Comment:
There is a library call mapstruct, try use it. It will save your time.
org.mapstruct
example:
`
import org.mapstruct.Mapper;
import org.mapstruct.Mapping;
import org.mapstruct.Named;

@Mapper(componentModel = "spring")
public interface ProductMapper 
    ProductBody toDomain(ProductBodyRequest request);
`
It will generate code for you